biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -320,8 +320,21 @@ public void prepare() throws Exception {
 						continue;
 					}
 
-					if (!dir.isDirectory()) {
-						IO.mkdirs(dir);
+					if (!dir.exists()) {
+						try {
+							IO.mkdirs(dir);
+						} catch (Exception ex) {
+							error("could not create src directory (in src property) %s: %s", dir, ex)
```

Comment:
When you have an exception object, use the `exception` method instead of `error`.